xinference/model/embedding/vllm/core.py

```diff
@@ -92,8 +103,12 @@ def create_embedding(
                 sentences = truncated_sentences[0]
             else:
                 sentences = truncated_sentences
-
-        outputs = self._model.embed(sentences, use_tqdm=False)
+        pool_params = PoolingParams(
+            dimensions=dimensions, normalize=normalize_embedding
```

Comment:
`normalize` seems a new option added in https://github.com/vllm-project/vllm/releases which released in vLLM 0.10.1, can you consider the compatibility for the users who still uses older vllm?